CHANGES.md

```diff
@@ -36,7 +36,9 @@ Change Log
 * Added support for vertex shader uniforms when `tileset.colorBlendMode` is  `MIX` or `REPLACE`. [#5874](https://github.com/AnalyticalGraphicsInc/cesium/pull/5874)
 * Added separate file for the Cesium [Code of Conduct](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CODE_OF_CONDUCT.md). [#6129](https://github.com/AnalyticalGraphicsInc/cesium/pull/6129)
 * Fixed applying a translucent style to a point cloud tileset. [#6113](https://github.com/AnalyticalGraphicsInc/cesium/pull/6113)
-* Fixed Sandcastle Particle System example for better visual [#6132](https://github.com/AnalyticalGraphicsInc/cesium/pull/6132)
+* Fixed sandcastle Particle System example for better visual [#6132](https://github.com/AnalyticalGraphicsInc/cesium/pull/6132)
+* Modified move forward and backward camera functions to allow for proper zooming for a 2D camera. [#6133](https://github.com/AnalyticalGraphicsInc/cesium/pull/6133)
```

Comment:
Link to the bug your fixing in this case, #5884, I think it's more relevant. I think you can compress both these bullets to `Fixed camera movement and look functions for 2D mode.`